public/locales/mr/common.json

```diff
@@ -49,7 +49,8 @@
     "CLOSE": "बंद",
     "CANCEL": "रद्द करा",
     "OK": "ठीक आहे",
-    "FROM_TO_DATE": "पासून - तारखा"
+    "FROM_TO_DATE": "पासून - तारखा",
```

Comment:
@Rushikesh-Sonawane99 its not correct translation

Reply:
@itsvick Modified translations